pinot-controller/src/main/resources/app/components/Table.tsx

```diff
@@ -292,6 +292,14 @@ export default function CustomizedTables({
     setPage(newPage);
   };
 
+  React.useEffect(() => {
+    if(isPagination) {
```

Comment:
(minor)
```suggestion
    if (isPagination) {
```

Reply:
This change is removed, it is no longer relevant